enumcover.go

```diff
@@ -80,29 +81,19 @@ func checkConsts(pass *analysis.Pass, n ast.Node, typeName string) {
 				case *ast.BasicLit:
 					namesForType[unquote(n.Value)] = true
 				case *ast.Ident:
-					if n.Obj != nil {
```

Comment:
With the change introduced in this commit. There won't be need for this conditions, they would be redundant if we keep them.